torch/csrc/jit/serialization/export.cpp

```diff
@@ -598,6 +609,33 @@ void EncoderBase::AddInitializersIntoGraphProto(
   }
 }
 
+size_t EncoderBase::GetGraphProtoSizes(
+    onnx::GraphProto* graph_proto,
+    const Block* block,
+    const std::map<std::string, at::Tensor>& initializers) {
+  size_t sizes = 0;
+  for (auto input : block->inputs()) {
+    auto name_tensor_pair = initializers.find(input->debugName());
+    if (name_tensor_pair == initializers.end()) {
+      continue;
+    }
+    onnx::GraphProto* graph_proto_copy = new onnx::GraphProto(*graph_proto);
+    auto tensor_proto = graph_proto_copy->add_initializer();
+    const at::Tensor tensor = name_tensor_pair->second;
+    at::Tensor t;
+    if (tensor.is_quantized()) {
+      t = tensor.contiguous();
+    } else {
+      t = tensor.contiguous().cpu();
+    }
+    tensor_proto->set_raw_data(std::string(
+        static_cast<char*>(t.data_ptr()), t.element_size() * t.numel()));
+    sizes += tensor_proto->ByteSizeLong();
```

Comment:
The size is computed based on the initializer tensor size. `GraphProto` contains other elements, do we also need consider that into the size?

Reply:
I try to change some elements like dimmension and data_type, but it seems not affect the model size. You means there may have some other elements?